R/MsExperiment.R

```diff
@@ -0,0 +1,63 @@
+#'@include PlainTextParam.R
+#'@title Methods to save and load contents of a MsExperiment object
+#'
+#' @description
+#'
+#' For `MsExperiment`, the exported data and related text files are:
+#'
+#' - The backend's [spectraData()] stored in a tabular format in a text file
+#'   named *backend_data.txt*.
+#'
+#' - The `processingQueueVariables`, `processing`, `processingChunkSize()` and
```

Comment:
hm, I don't think you should describe these files here - they are specific for `Spectra`, no? I would then maybe link here to the documentation of the `saveMsObject,Spectra` instead and document them here. So, something along *for a description of the exported data files representing the content of the object's `Spectra` object see ...*

Reply:
yes so to be honest i was thinking of grouping everything in the PlaintTextParam documentation

Reply:
hm, yes, makes also sense - so we don't have documentation scattered around. but then I would maybe suggest to have them separated by section maybe? something like

```r

#' @section On disk storage for `MsBackendMzR` objects:
#'
#' For `MsBackendMzR` objects the following files get stored:
#' 
#' ....
#'
#' @section On disk storage for `Spectra` objects:
```

this would also make it easier to extend the documentation later. And in the section for `Spectra` you could refer to the `MsBackendXXX` section(s) for details of their stored file(s).

Reply:
i agree, but i will keep the previous description that is not rendered in the object specific .R files, mostly to help developers. Is that okay ?